frontend/src/store/objectStore.ts

```diff
@@ -73,15 +73,17 @@ export const useObjectStore = defineStore('objectStore', () => {
         const uniqueIds = [...new Set(permResponse.map((x: { objectId: string }) => x.objectId))];
         const objects = (await objectService.listObjects({ objId: uniqueIds, ...params })).data;
         const metadataResponse = (await objectService.getMetadata(null, { objId: uniqueIds })).data;
+        const objectTaggingResponse = (await objectService.getObjectTagging()).data;
```

Comment:
We might need to think about how much related data we call in for potentially all object here.
I guess an advantage to getting all the data here is that it should be available in the store when you visit the file details page. but i was expecting another separate call there.

separate note: We plan to alter the COMS /object search endpoint soon.. it will factor in user permissions so this function could be simplified a bit.